pkg/security/pem.go

```diff
@@ -100,6 +100,9 @@ func PEMToCertificates(contents []byte) ([]*pem.Block, error) {
 		var block *pem.Block
 		block, contents = pem.Decode(contents)
 		if block == nil {
+			if len(contents) > 0 {
+				return nil, errors.Errorf("block #%d could not be parsed", len(certs))
```

Comment:
This check is stricter than it used to be. If we still want to make use of the certificates seen so far, we could just log a warning.